src/types.ts

```diff
@@ -160,7 +160,7 @@ function getComplexType(
         (entry: any) => {
           const typeDecl = get(ast, entry.value, importedPropTypes, options);
           return dom.create.property(
-            entry.key.name,
+            entry.key.type === 'StringLiteral' ? `${entry.key.value}` : entry.key.name,
```

Comment:
object structure of `entry.key` for nested `'aria-label'`:

```json
  {
    "type": "StringLiteral",
    "start": 3592,
    "end": 3604,
    "loc": { "start": { "line": 133, "column": 4 }, "end": { "line": 133, "column": 16 } },
    "extra": { "rawValue": "aria-label", "raw": "'aria-label'" },
    "value": "aria-label"
  },
```

and object structure of `entry.key` for regular prop (`content` in this case):
```json
  {
    "type": "Identifier",
    "start": 3639,
    "end": 3646,
    "loc": {
      "start": { "line": 134, "column": 4 },
      "end": { "line": 134, "column": 11 },
      "identifierName": "content"
    },
    "name": "content"
  }
```